tests/h/api/views_test.py

```diff
@@ -73,8 +73,8 @@ def test_it_searches(self, pyramid_request, search_lib):
                                                   separate_replies=False)
 
     def test_it_loads_annotations_from_database(self, pyramid_request, search_lib, storage):
-        search_lib.search.return_value = {'total': 2,
-                                          'rows': [{'id': 'row-1'}, {'id': 'row-2'}]}
+        search_lib.search.return_value = mock.Mock(total=2,
```

Comment:
Given that `SearchResult` is just a data structure, it might make more sense to use it here in place of a `Mock` object.